cmake/SNNHelpers.cmake

```diff
@@ -259,7 +259,7 @@ function(snn_executable)
   # Allows for resolved symbols in executables, which is required for debug
   # builds, where some unreachable references to undefined functions don't get
   # dead code eliminated.
-  if(${CMAKE_BUILD_TYPE} STREQUAL "Debug")
+  if(${CMAKE_BUILD_TYPE} STREQUAL "Debug" AND NOT MSVC)
```

Comment:
Again, this will be removed with the internal changes. Please wait for that to be merged.